test/load/allocation/grpc/fleet.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: load-test-fleet
+spec:
+  # the number of GameServers to keep Ready or Allocated in this Fleet
+  replicas: 4000
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  template:
+    metadata:
+      labels:
+        foo: bar
+    # GameServer specification
+    spec:
+        health: {}
```

Comment:
```suggestion
```

Nit: this isn't doing anything here either, so can be removed.

Reply:
if we are going to cleanup, would be good to get rid of this too :+1:

Reply:
Fixed